app/client/src/components/shared/CharacteristicsSelect.tsx

```diff
@@ -1,41 +1,57 @@
-import uniqueId from 'lodash/uniqueId';
-import { useContext, useMemo, useState } from 'react';
+import { useContext, useMemo } from 'react';
```

Comment:
Related to below, I think it's better to keep the visible label.
```suggestion
import uniqueId from 'lodash/uniqueId';
import { useContext, useMemo, useState } from 'react';
```